tests/test_minimum_clearance.py

```diff
@@ -0,0 +1,33 @@
+"""
+Tests for the minimum clearance property.
+"""
+
+import math
+
+import pytest
+
+from shapely.wkt import loads as load_wkt
+from shapely.geos import geos_version
+
+
+pytestmark = pytest.mark.skipif(geos_version < (3, 6, 0), reason='GEOS 3.6.0 required')
+
+
+def test_point():
```

Comment:
@tomplex I've checked out your branch and when I modify it to decorate the new test functions as in the suggestion below, the skipping seems to work as intended. Note: I have GEOS 3.8.0, so I also modified `pytestmark` to skip if the version < 3.9.0 in my test.

```suggestion
@pytestmark
def test_point():
```